modules/react/layout/lib/utils/grid.ts

```diff
@@ -1,4 +1,7 @@
 import {Property} from 'csstype';
+import {space as spaceTokens, CanvasSpace, CanvasSpaceKeys} from '@workday/canvas-kit-react/tokens';
+
+type GridSpacePropsValues = CanvasSpaceKeys | number | (string & {});
```

Comment:
I _think_ we'll need to export this. I'll have to pull it down locally to know for sure.

Reply:
Can we add an export here?

Reply:
I'll add an export